src/lib/components/cards/cardButton/index.tsx

```diff
@@ -19,17 +19,21 @@ type ActionProps =
 const CardContent = ({
   title,
   description,
-  maxChar,
+  ellipsis,
 }: {
   title: string;
   description: string;
-  maxChar?: number;
+  ellipsis: boolean;
 }) => (
   <>
-    <div>
+    <div className="w90">
       <div className="p-p--small">{title}</div>
-      <div className="tc-primary-500 p-p">
-        {maxChar ? limitTextLength(description, maxChar) : description}
+      <div
+        className={classnames('tc-primary-500 p-p', {
+          [styles['text-ellipsis']]: ellipsis,
+        })}
+      >
+        {description}
```

Comment:
What about doing another approach here? Instead of having `ellipsis: boolean;` you can accept description as either `string` or `React.ReactNode`. If it's of style string you render it with `p-p tc-primary-500` and if it's of type node you just render the component. It's a bit more flexible I would say as you live the ellipsis implementation to the parent component.

Wdyt?

Reply:
I'm not sure I understand what you're trying to achieve, why would we need a JSX type of `description` prop if the parent will be implemented with ellipsis anyway? Could you give an example?